projects/aca-content/src/lib/components/search/search-save/list/smart-list/saved-searches-smart-list.component.spec.ts

```diff
@@ -0,0 +1,87 @@
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { CORE_PIPES, CoreTestingModule } from '@alfresco/adf-core';
+import { BehaviorSubject, ReplaySubject, Subject } from 'rxjs';
+import { By } from '@angular/platform-browser';
+import { SavedSearchesService } from '@alfresco/adf-content-services';
+import { SavedSearchesSmartListComponent } from './saved-searches-smart-list.component';
+import { SavedSearch } from '@alfresco/adf-content-services/lib/common/interfaces/saved-search.interface';
+import { AppService, DocumentBasePageService, DocumentBasePageServiceMock } from '@alfresco/aca-shared';
+import { provideMockStore } from '@ngrx/store/testing';
+
+const appServiceMock = {
+  appNavNarMode$: new BehaviorSubject('collapsed'),
+  setAppNavbarMode: jasmine.createSpy('setAppNavbarMode'),
+  toggleAppNavBar$: new Subject()
+};
+
+describe('SavedSearchesSmartListComponent', () => {
+  let fixture: ComponentFixture<SavedSearchesSmartListComponent>;
+  let fakeSavedSearches$: ReplaySubject<SavedSearch[]>;
+
+  beforeEach(() => {
+    fakeSavedSearches$ = new ReplaySubject<SavedSearch[]>(1);
+    TestBed.configureTestingModule({
+      imports: [CoreTestingModule, SavedSearchesSmartListComponent],
+      providers: [
+        provideMockStore(),
+        ...CORE_PIPES,
+        { provide: DocumentBasePageService, useClass: DocumentBasePageServiceMock },
+        { provide: SavedSearchesService, useValue: { savedSearches$: fakeSavedSearches$ } },
+        { provide: AppService, useValue: appServiceMock }
+      ]
+    });
+
+    fixture = TestBed.createComponent(SavedSearchesSmartListComponent);
+  });
+
+  it('should show the list of searches', async () => {
+    const mockSavedSearches: SavedSearch[] = [
+      { name: '1', order: 0, encodedUrl: '' },
+      { name: '2', order: 1, encodedUrl: '' }
+    ];
+    fakeSavedSearches$.next(mockSavedSearches);
+    fixture.detectChanges();
+    const listComponent = fixture.nativeElement.querySelector('aca-saved-searches-ui-list');
+    expect(listComponent).toBeDefined();
+  });
+
+  it('should show the no content template when no users are found', async () => {
+    const mockSavedSearches: SavedSearch[] = [];
+    fakeSavedSearches$.next(mockSavedSearches);
+    fixture.detectChanges();
+
+    const emptyContent = fixture.nativeElement.querySelector('.adf-empty-content');
+    expect(emptyContent).toBeDefined();
+  });
+
+  it('should show the spinner while user list is loading', async () => {
```

Comment:
Should this be  "should show the spinner while _**saved searches**_ list is loading" ?